README.MD

```diff
@@ -19,13 +19,39 @@ see simple-example.py
 pip install er-client
 ```
 
+## Usage
 Then in your code, import the library and create an instance of the client.
 
 ```
 from erclient import ERClient
 
 client = ERClient(service_root="https://sandbox.pamdas.org/api/v1.0", username="", password="")
 ```
+## Async Support
+We also offer an async client (asyncio).
+
+Disclaimer: The async client is experimental and the current capabilities are limited to:
+* Posting Sensor Observations (a.k.a Positions)
+* Posting Reports (a.k.a Events)
+* Posting Camera Trap Reports
+```
+from erclient import AsyncERClient
+
+# You can use it as an async context-managed client
+async with AsyncERClient(service_root="https://sandbox.pamdas.org/api/v1.0", username="", password="") as client:
+   await self.er_client.post_sensor_observation(position)
+   await client.post_report(report)
+   await self.er_client.post_camera_trap_report(camera_trap_payload, file)
+   ...
+
+# Or create an instance and close the client explicitly later
+client = AsyncERClient(service_root="https://sandbox.pamdas.org/api/v1.0", username="", password="") as client:
```

Comment:
These use examples are really great.

Remove `as client` from this line.

Reply:
ohh good catch. I copy/pasted from the other example and I forgot to delete that. Thank you.